app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -215,6 +215,17 @@
 
 .gem-c-image-card__youtube-video-embed iframe {
   z-index: 1;
+  transform: scale(0.7);
```

Comment:
This is a good solution, but I think it needs some tweaking. On both mobile and desktop it looks a little misaligned, although more noticeably on mobile.

On desktop it's not quite within the parent element (not sure if this is a problem, I haven't seen it alongside normal page content).

![Screenshot 2023-01-11 at 14 22 20](https://user-images.githubusercontent.com/861310/211830522-53367f72-0559-4538-bdef-5476f503f1d1.png)

On mobile it's too far to the right.

![Screenshot 2023-01-11 at 14 22 37](https://user-images.githubusercontent.com/861310/211830547-26ae13f7-f375-459b-ba34-c66bbc4dc044.png)

Reply:
have fixed these two layout issues

Reply:
Nice!